### PR TITLE
fix(jsonschema): unknown object type should resolve to dict arbitrary

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #48** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/48" target="_blank">Graphite</a> 
    * **PR #49** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/49" target="_blank">Graphite</a> 
      * **PR #50** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/50" target="_blank">Graphite</a> 
        * **PR #51** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/51" target="_blank">Graphite</a> 
          * **PR #52** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/52" target="_blank">Graphite</a> 
            * **PR #53** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/53" target="_blank">Graphite</a> 
              * **PR #54** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/54" target="_blank">Graphite</a> 
                * **PR #55** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/55" target="_blank">Graphite</a> 
                  * **PR #56** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/56" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/56?utm_source=stack-comment).

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #48** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/48" target="_blank">Graphite</a> 
    * **PR #49** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/49" target="_blank">Graphite</a> 
      * **PR #50** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/50" target="_blank">Graphite</a> 
        * **PR #51** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/51" target="_blank">Graphite</a> 
          * **PR #52** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/52" target="_blank">Graphite</a> 
            * **PR #53** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/53" target="_blank">Graphite</a> 
              * **PR #54** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/54" target="_blank">Graphite</a> 
                * **PR #55** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/55" target="_blank">Graphite</a> 
                  * **PR #56** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/56" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/56?utm_source=stack-comment).

File: src/lib/primitives/jsonschema/jsonschema.ts [hunk 1]
```diff
@@ -123,6 +124,9 @@ const schemaWalker: JsonSchemaWalker = {
                 (isBoolean(node.additionalProperties) && keysOf(node.additionalProperties).length === 0)) &&
```
Comment (MickVanDuijn) [inline src/lib/primitives/jsonschema/jsonschema.ts:124]: <i class='graphite__hidden'>[Re: lines 124 to 124]<br/></i>

don't we have a clash between `isBoolean` and `keysOf` here? :thinking: I would expect a negation here at first glance (or maybe I need a coffee first)<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/skyleague/therefore/56?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/56?utm_source=stack-comment).


Job Reference: G9qW3Gh7LcbjHSuwHmyz

**Pull request merged**